src/dcd_mapping/annotate.py

```diff
@@ -367,23 +375,34 @@ def annotate(
     """
     score_annotations = []
     for mapped_score in mapped_scores:
-        if isinstance(mapped_score.pre_mapped, Haplotype) and isinstance(
-            mapped_score.post_mapped, Haplotype
+        if mapped_score.pre_mapped is None:
+            score_annotations.append(
+                ScoreAnnotationWithLayer(
+                    mavedb_id=mapped_score.accession_id,
+                    score=float(mapped_score.score) if mapped_score.score else None,
+                    error_message=mapped_score.error_message,
+                )
+            )
+        elif isinstance(mapped_score.pre_mapped, Haplotype) and (
+            isinstance(mapped_score.post_mapped, Haplotype)
+            or mapped_score.post_mapped is None
         ):
             score_annotations.append(
                 _annotate_haplotype_mapping(
                     mapped_score, tx_results, metadata, vrs_version
                 )
             )
-        elif isinstance(mapped_score.pre_mapped, Allele) and isinstance(
-            mapped_score.post_mapped, Allele
+        elif isinstance(mapped_score.pre_mapped, Allele) and (
+            isinstance(mapped_score.post_mapped, Allele)
+            or mapped_score.post_mapped is None
         ):
             score_annotations.append(
                 _annotate_allele_mapping(
                     mapped_score, tx_results, metadata, vrs_version
                 )
             )
         else:
+            # TODO how to combine this error message with other potential error messages?
             ValueError("inconsistent variant structure")
```

Comment:
Not sure if these will be other exceptions, or just messages, but we could try doing something with exception groups if this was still an issue: https://docs.python.org/3/tutorial/errors.html#raising-and-handling-multiple-unrelated-exceptions

Also, is this missing a `raise` or is that intentional?